llarp/router/router.cpp

```diff
@@ -1033,13 +1018,20 @@ namespace llarp
     return _link_manager.get_random_connected(result);
   }
 
+  std::unordered_set<RouterID>
+  Router::router_whitelist() const
+  {
+    return _node_db->whitelist();
```

Comment:
Two things:
1) Maybe less minor: `NodeDB::whitelist()` returns a const reference to the unordered set, so should the invoking `Router::router_whitelist()` method do the same?
2) Super minor: I feel like calling it `Router::router_whitelist()` is redundant, maybe we just call the getters after the object? Like `Router::get_whitelist()` and so on. I know `get_{whatever}` is a little on the nose, but a few lines down you call `NodeDB::set_router_whitelist()`, which is very descriptive and good